packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -89,13 +92,27 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// The bottom sheet's background color.
+  ///
+  /// Defines the bottom sheet's [Material.color].
+  ///
+  /// Defaults to [Colors.white].
+  final Color color;
+
   /// The z-coordinate at which to place this material relative to its parent.
   ///
   /// This controls the size of the shadow below the material.
   ///
   /// Defaults to 0. The value is non-negative.
   final double elevation;
 
+  /// The shape of the bottom sheet's [Material].
+  ///
+  /// Defines the bottom sheet's [Material.shape].
+  ///
+  /// Defaults to null.
```

Comment:
For `color`, even though the default is null, you say what `Material` will do when passed null. I get that it's kind of a courtesy, but you should be consistent. 

For both you should either say `Defaults to null`, or `If null, [Material] will default to Colors.white (or whatever shape)`.

Reply:
Done